config/tg-notifier.php

```diff
@@ -9,6 +9,8 @@
             'url'      => $_ENV['TGN_APP_URL'] ?? 'http://localhost:3000',
 
             'timezone' => $_ENV['TIMEZONE'] ?? 'Asia/Ho_Chi_Minh',
+
+            'verify_system' => $_ENV['VERIFY_SYSTEM'] ?? 'true',
```

Comment:
@LuongTienThinh Please change the key to `request_verify` and the environment name the `TGN_REQUEST_VERIFY`

Reply:
done